source/FreeRTOS_TCP_Transmission.c

```diff
@@ -1474,7 +1474,7 @@
                      * 'ulSequenceNumber' and 'ulAckNr' will be swapped. */
                     uint32_t ulSequenceNumber = FreeRTOS_ntohl( pxTCPPacket->xTCPHeader.ulSequenceNumber );
                     ulSequenceNumber++;
-                    pxTCPPacket->xTCPHeader.ulSequenceNumber = FreeRTOS_ntohl( ulSequenceNumber );
+                    pxTCPPacket->xTCPHeader.ulSequenceNumber = FreeRTOS_htonl( ulSequenceNumber );
```

Comment:
`ntohl` versus `htonl` : as you know, their math is the same, but the semantics are different.
Thank you!

Reply:
Yes, internally, both the functions are same. But, for the sake of consistency, I changed this as we are putting data into the network buffer which should be in network format.
Thank you @htibosch for this PR.